Foundation/NSCalendar.swift

```diff
@@ -163,7 +163,10 @@ public class NSCalendar : NSObject, NSCopying, NSSecureCoding {
     }
     
     public func copyWithZone(_ zone: NSZone) -> AnyObject {
-        NSUnimplemented()
+        let copy = NSCalendar(calendarIdentifier: calendarIdentifier)!
+        copy.locale = locale
+        copy.timeZone = timeZone
+        return copy
```

Comment:
ah, right! I missed mutation of those fields. I've changed implementation to consider above point and added unit testcase to explicitly check those fields.